command/volume_stats.go

```diff
@@ -324,29 +324,29 @@ func NewControllerClient(address string) (*ControllerClient, error) {
 	}, nil
 }
 
-// GetStatus will return json response and statusCode
-func GetVolumeStats(address string, obj interface{}) (error, int) {
+// GetStatuStats will return json response and statusCode
```

Comment:
@keydrevolta can you fix the comments please

Reply:
Done :D